test/libyul/yulOptimizerTests/loadResolver/keccak_reuse_expr_mstore.yul

```diff
@@ -0,0 +1,28 @@
+{
+    let a := calldataload(0)
+    sstore(f(keccak256(0, a)), keccak256(0, a))
+    sstore(f(keccak256(0, a)), keccak256(0, a))
```

Comment:
Shouldn't we also test the opposite case, i.e.
``sstore(keccak256(0, a), f(keccak256(0, a)))``
?